mmv1/products/compute/api.yaml

```diff
@@ -6561,6 +6561,14 @@ objects:
                     disks.source is required.  To create a disk with one of
                     the public operating system images, specify the image
                     by its family name.
+                - !ruby/object:Api::Type::KeyValuePairs
```

Comment:
You can revert the changes to this file- they don't do anything, as MMv1 doesn't generate this resource (although it's used to handle the copied files)